internal/dataplane/parser/translate_utils.go

```diff
@@ -86,6 +86,7 @@ func isRefAllowedByPolicy(
 // ReferencePolicy with those Tos and the input From.
```

Comment:
I believe this godoc has to be updated as well, right?

Reply:
Yep, I updated it.